artcommon/artcommonlib/konflux/konflux_db.py

```diff
@@ -58,11 +58,21 @@ def add_build(self, build: konflux_build_record.KonfluxBuildRecord):
         Insert a build record into Konflux DB
         """
 
+        def value_or_null(value):
```

Comment:
`stringify_val` or something like that might be a better name?

Reply:
Here I am taking the value and casting to a proper value that BigQuery can digest. Strings need to be quoted in `''`, other values don't. For example, timestamps are rendered inside the query string without quotes. `None` values translate into NULL. `stringify_val` does not tell it all

Reply:
ah okay! Then it makes sense. Can we include this in a comment? would make it clear why we're doing this

Reply:
Done in 9717adcbe18487dc4954490de5fbc4f6201be5bd